omnibus/config/software/inspec.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# All Rights Reserved.
+#
+
+name 'inspec'
+
+dependency 'ruby'
+dependency 'train'
+
+license "Apache-2.0"
+license_file "LICENSE"
+
+default_version "v0.18.0"
```

Comment:
The challenge is, that I want to be able to built exactly the package for the sha I've checked out